Kaden's "hacker card"

ofluffydev opened the pull request:
## Submission Checklist:

- [x] I am a current high school, middle school, or home schooled student.
  - [x] I have filled out the [verification form](https://airtable.com/shrbcDANe6ug6gj0p)
- [x] I have joined the [`#onboard`](https://hackclub.slack.com/archives/C056AMWSFKJ) channel on Slack
- [x] I made this board from scratch, even if I followed a tutorial
- [x] I have followed [DIRECTIONS.md](https://github.com/hackclub/OnBoard/blob/main/DIRECTIONS.md)
  - [x] Created a folder under `onboard/projects`
  - [x] Filled out `TEMPLATE.md` as `README.md`
  - [x] Uploaded Gerber, sources, and schematic
  - [x] Uploaded screenshot of PCB order with all costs included
  - [x] I am ordering the smallest batch size (ie. JLCPCB's smallest order is for 5 boards)
  - [x] If outside the US I've checked that I can afford the customs charges in my country (which isn't covered by the grant)

- [x] (Optional) This project is from a tutorial[^1].

[^1]: Projects from a tutorial are 100% fine! We just want to ask so we can count how people are using tutorials.

<!-- -Submission- -->

daisybanks reviewed (approved):
Welcome OnBoard @ofluffydev! Everything looks good on HCB's side @hackclub/boarding-party. :)

karmanyaahm reviewed (changes requested):
Love the design! However, you have not selected assembly from JLCPCB, so there won't be any components installed on your board. Please check out https://github.com/hackclub/OnBoard/blob/main/docs/ordering_from_JLCPCB.md#pcb-assembly for more info.

Additionally, the src folder should also have a JSON from the schematic page, just like the PCB page.

ofluffydev commented:
> Love the design! However, you have not selected assembly from JLCPCB, so there won't be any components installed on your board. Please check out https://github.com/hackclub/OnBoard/blob/main/docs/ordering_from_JLCPCB.md#pcb-assembly for more info.
> 
> Additionally, the src folder should also have a JSON from the schematic page, just like the PCB page.

Added those, did what you said and updated the price to match it.

karmanyaahm reviewed (approved):
LGTM, sorry for the delay.
